packages/ses/src/error/assert.js

```diff
@@ -214,21 +226,21 @@ const tagError = (err, optErrorName = err.name) => {
  */
 const makeError = (
   optDetails = redactedDetails`Assert failed`,
-  ErrorConstructor = Error,
+  ErrorConstructor = FERAL_ERROR,
```

Comment:
Since we're exempting `assert` from most of this anyway, could we have it instead use the current binding of `globalThis.Error`? That way, after lockdown, it would use a safer one.
```suggestion
  ErrorConstructor = globalThis.Error,
```